base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -34,6 +34,29 @@ static BOOL force_termination = FALSE;
 static WCHAR **task_list;
 static unsigned int task_count;
 
+static WCHAR opForceTerminate[] = {'f',0};
+static WCHAR opImage[] = {'i','m',0};
+static WCHAR opPID[] = {'p','i','d',0};
```

Comment:
If code is not wine-synced you can use actually readable wide strings:
`static WCHAR opPID[] = L"pid";`

Reply:
Actually I don't know why writing like this.
the origin code looks like this and I just copy it ...
Is there anything to do with wine?